asapdiscovery-dataviz/asapdiscovery/dataviz/html_viz.py

```diff
@@ -110,12 +115,14 @@ def __init__(
             else:
                 self.logger.warning(f"Pose {pose} does not exist, skipping.")
 
-        if not protein.exists():
-            raise ValueError(f"Protein {protein} does not exist.")
-
-        self.protein = openeye_perceive_residues(
-            load_openeye_pdb(str(protein)), preserve_all=True
-        )
+        if isinstance(protein, oechem.OEMolBase):
+            self.protein = protein.CreateCopy()
+        else:
+            if not protein.exists():
+                raise ValueError(f"Protein {protein} does not exist.")
+            self.protein = openeye_perceive_residues(
+                load_openeye_pdb(str(protein)), preserve_all=True
```

Comment:
`load_openeye_pdb` can take a `Path` as input now
```suggestion
                load_openeye_pdb(protein), preserve_all=True
```